scanners/analytics.py

```diff
@@ -22,8 +21,7 @@ def init(environment, options):
         analytics_path = os.path.join(utils.cache_dir(), "analytics.csv")
 
         try:
-            response = requests.get(analytics_file)
-            utils.write(response.text, analytics_path)
+            utils.download(analytics_file, analytics_path)
```

Comment:
The `analytics` scanner was the only place `requests` was being used, and it was easy to replace with our other download method.